Library/ENSideMenu.swift

```diff
@@ -459,6 +459,10 @@ public class ENSideMenu : NSObject, UIGestureRecognizerDelegate {
     }
     
     private func updateSideMenuApperanceIfNeeded () {
+        if let superview = sideMenuContainerView.superview {
```

Comment:
No need to unwrap here, can just use the optional `sideMenuContainerView.superview?`.